### PR TITLE
repositories merge

### PR DESCRIPTION
**Pull request by avelino**
fixed: #361

**Inside the pREST repository to keep history (commits credit)**
```
git clone git@github.com:prest/transactions.git
cd transactions
mkdir transactions
mv * transactions
git commit -m "Moved entire subrepo into transactions"
git fetch transactions
rm -rf transactions/transactions
rm -rf transactions/.git
```

_operation generated for all sub repositories_

### CODE REVIEW

**Comment by avelino** [comment]
@tiaguinho check:
- https://app.sourcelevel.io/github/prest/prest/pulls/370?utm_campaign=prest&utm_medium=pr&utm_source=github
- travis-ci test environment

File: controllers/tables_test.go [hunk 1]
```diff
@@ -90,7 +90,7 @@ func TestSelectFromTables(t *testing.T) {
 		status      int
 		body        string
 	}{
-		{"execute select in a table with array", "/prest/public/testarray", "GET", http.StatusOK, "[{\"id\":100,\"data\":[\"Gohan\",\"Goten\"]}]"},
+		{"execute select in a table with array", "/prest/public/testarray?id=100", "GET", http.StatusOK, "[{\"id\":100,\"data\":[\"Gohan\",\"Goten\"]}]"},
```
Comment (avelino) [inline controllers/tables_test.go:93]: we need to write a test without passing **ID** `?id=100`

File: controllers/tables_test.go [hunk 2]
```diff
@@ -100,7 +100,7 @@ func TestSelectFromTables(t *testing.T) {
 		{"execute select in a table with order clause empty", "/prest/public/test?_order=", "GET", http.StatusOK, ""},
 		{"execute select in a table with custom where clause and pagination", "/prest/public/test?name=$eq.nuveo&_page=1&_page_size=20", "GET", http.StatusOK, ""},
 		{"execute select in a table with select fields", "/prest/public/test5?_select=celphone,name", "GET", http.StatusOK, ""},
-		{"execute select in a table with select *", "/prest/public/test5?_select=*", "GET", http.StatusOK, ""},
+		{"execute select in a table with select *", "/prest/public/test5?_select=*", "GET", http.StatusOK, "[{\"id\":1,\"name\":\"prest tester\",\"celphone\":\"444444\"}]"},
```
Comment (avelino) [inline controllers/tables_test.go:103]: we need to write a test where the return is **empty**
